test/unit/acl/acl.test.ts

```diff
@@ -22,230 +22,374 @@ import {
 import { AgentMapMap, ComboList } from '../../../src/acl/types'
 import { sym } from 'rdflib'
 import { createLiveStore } from '../helpers/createLiveStore'
+import { clearStore } from '../helpers/clearStore'
+import globalStore from '../../../src/store'
+import ns from '../../../src/ns'
+import {
+  ACL_LINK,
+  DEFAULT_CONTAINER_ACL,
+  DEFAULT_CONTAINER_DOC,
+  DEFAULT_RESOURCE_ACL,
+  DEFAULT_RESOURCE_DOC,
+  getAgentMapForAclWithAllAgents,
+  getAgentMapForAclWithAllAgentsUsingDefaults, mocks, PROFILE
+} from './acl.mocks'
+import { loadTurtleIntoGraph } from '../helpers/loadTurtleIntoGraph'
+import { createGraphFromTurtle } from '../helpers/createGraphFromTurtle'
+import { createAgentMapFromTurtle } from '../helpers/createAgentMapFromTurtle'
 
 jest.mock('solid-auth-client')
 
-describe('ACLbyCombination', () => {
-  it('exists', () => {
-    expect(ACLbyCombination).toBeInstanceOf(Function)
-  })
-  it('runs', () => {
-    expect(ACLbyCombination({} as AgentMapMap)).toBeInstanceOf(Object)
-  })
-})
-
-describe('ACLToString', () => {
-  it('exists', () => {
-    expect(ACLToString).toBeInstanceOf(Function)
-  })
-  it('runs', () => {
-    expect(ACLToString({} as AgentMapMap)).toEqual('{}')
-  })
-})
+// describe('ACLbyCombination', () => {
+//   it('exists', () => {
+//     expect(ACLbyCombination).toBeInstanceOf(Function)
+//   })
+//   it('runs', () => {
+//     expect(ACLbyCombination({} as AgentMapMap)).toBeInstanceOf(Object)
+//   })
+// })
+//
+// describe('ACLToString', () => {
+//   it('exists', () => {
+//     expect(ACLToString).toBeInstanceOf(Function)
+//   })
+//   it('runs', () => {
+//     expect(ACLToString({} as AgentMapMap)).toEqual('{}')
+//   })
+// })
```

Comment:
Why did you comment this out? Committed by accident?

Reply:
Sorry about that, forgot to readd tests that I didn't have time to complete. I've readded the tests in my latest commit.